### ISSUE TITLE
Add Self Signed Cert Issuer

### ISSUE THREAD

**Issue by ranchodeluxe** [developmentseed/eoapi-k8s#136]
### Description

# Goal
We should also include a self-signed cert issuer for every one to use

### Labels

- [X] enhancement
- [ ] discussion
- [ ] help wanted

**Comment by sunu** [comment]
Yes, documenting how to use cert-manager and lets encrypt in the docs should be good enough.

**Comment by pantierra** [comment]
This is done in [ingress documentation](https://github.com/developmentseed/eoapi-k8s/blob/main/docs/unified-ingress.md).
